packages/removable-tags/src/components/RemovableTags.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import StyledTag from "./styledComponent/StyledTag";
+
+class RemovableTags extends Component {
```

Comment:
Rename package to `Tag`.

Make tag "removable" by receiving a property that says it is removable.
If not removable, do not render the close button.